### PR TITLE
Admin - Print user table profiles

### LINKED ISSUE #2644: Download multi-users as text files
<sup>ℹ️ [Figma (root)][figma] | [Sitemap][sitemap]</sup>
<sup>_This initial comment is collaborative and open to modification by all._</sup>

## Screenshot: Prototype
[![a screenshot of ...][image]][image-link]
<sup>[Click screenshot to visit Figma.][image-link]</sup>

## Acceptance Criteria
- [ ] Must be able to select multiple rows in table
- [ ] Clicking Download Profiles button generates a page for printing which lists all selected profiles, based on the same strategy used on the Admin User view.  

   [image]: https://user-images.githubusercontent.com/305339/166574896-aaf72687-608e-412b-87f7-07808ea5964e.png
   [image-link]: https://www.figma.com/proto/XS4Ag6GWcgdq2dBlLzBkay/Admin-v3?node-id=465%3A8698&scaling=min-zoom&page-id=0%3A1&starting-point-node-id=410%3A1984&hide-ui=1
   [figma]: https://go.talent.c4nada.ca/figma
   [sitemap]: https://go.talent.c4nada.ca/sitemap

### PR DESCRIPTION
**Pull request by mnigh**
Resolves #2644. h/t @esizer.

### Bonus

- [updates Loading component padding, background color](https://github.com/GCTC-NTGC/gc-digital-talent/commit/9dc64ec258173407ebc0591f42db8ccaf9f39ed9)

### Steps to Test
1. Refresh containers `docker-compose run --rm maintenance bash refresh_all.sh`
2. Navigate and login as admin user at `/admin`
3. Navigate to Users Table `/en/admin/users`
4. Slect all or a few of the rows using the checkbox on the far right column
6. Scroll to table footer at the bottom
7. Near **Selected actions** click **Print** button
8. See print preview screen with rows of users selected

### CODE REVIEW

File: frontend/admin/src/js/components/user/UserProfileDocument.tsx [hunk 1]
```diff
@@ -104,115 +100,122 @@ export const UserProfileDocument = React.forwardRef<
...
+                    </Heading>
+                  </HeadingWrapper>
+                  <ExperienceByTypeListing
+                    experiences={applicant.experiences?.filter(notEmpty)}
+                    defaultOpen
+                  />
+                </div>
+              );
+            })}
+          ;
```
Comment (vd1992) [inline frontend/admin/src/js/components/user/UserProfileDocument.tsx:218]: I think this is might be an accident
and is probably causing this orphaned semicolon here

image

**Comment by vd1992** [comment]
very clever reworking the other print document to fit single or multiple users, enables consistency too
elegant solution!

also
maybe a small suggestion, not important, but as a quality of life (QoL) improvement can it be more easy to distinguish where one user ends and another begins? for when you are printing a bunch of users at once you could easily push to 15, 30 pages
doesn't need to be a new page, maybe a divider line

image

**Review by petertgiles — CHANGES_REQUESTED** [review]
Awesome!  It's very performant, too.  I like the loader over the print button while fetching, especially.  I agree with Vachan that some more visible separation between applicants would be good.

**Review by petertgiles — APPROVED** [review]
Love it!

**Pull request merged**